spec.html

```diff
@@ -1505,7 +1537,7 @@ <h1>Number::bitwiseNOT ( _x_ )</h1>
 
         <emu-clause id="sec-numeric-types-number-exponentiate" oldids="sec-applying-the-exp-operator">
           <h1>Number::exponentiate ( _base_, _exponent_ )</h1>
-          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-dependent approximation of the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
+          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-approximated result to raising _base_ to the power _exponent_, subject to the following requirements:</p>
```

Comment:
This reads a bit strangely to me. Perhaps

```suggestion
          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-approximated value representing the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
```

(Same comment everywhere else this particular wording occurs, of course.)

Reply:
Done for all sites.